pennylane/ops/functions/equal.py

```diff
@@ -25,12 +26,11 @@
 def equal(
     op1: Union[Operator, MeasurementProcess, ShadowMeasurementProcess],
     op2: Union[Operator, MeasurementProcess, ShadowMeasurementProcess],
```

Comment:
Realizing that we don't need to specify `ShadowMeasurementProcess` here, since it is a `MeasurementProcess`.